plugin/db/util/driverutil.go

```diff
@@ -281,9 +281,9 @@ func endMigration(ctx context.Context, l *zap.Logger, executor MigrationExecutor
 }
 
 // Query will execute a readonly / SELECT query.
-func Query(ctx context.Context, l *zap.Logger, db *sql.DB, statement string, limit int) ([]interface{}, error) {
+func Query(ctx context.Context, l *zap.Logger, sqldb *sql.DB, statement string, limit int) ([]interface{}, error) {
```

Comment:
why changing this?

Reply:
I used to want to declare a struct from `github.com/bytebase/bytebase/plugin/db`, but undefined error. So I change it with the same format with https://github.com/bytebase/bytebase/blob/main/plugin/db/util/driverutil.go#L98